tests/st/manifests/mock-node-second.yaml

```diff
@@ -0,0 +1,14 @@
+# This is a Node used by the k8s ST tests.  The migration tests
```

Comment:
Comment incorrect for this new file.

Reply:
Should I update the other one as well? it's also incorrect

Reply:
Oh wait, no.. it's still true (applies to migration tests in ST)